layers/dot11.go

```diff
@@ -926,6 +926,7 @@ func (m *Dot11MgmtAssociationReq) NextLayerType() gopacket.LayerType {
 func (m *Dot11MgmtAssociationReq) DecodeFromBytes(data []byte, df gopacket.DecodeFeedback) error {
 	m.CapabilityInfo = binary.LittleEndian.Uint16(data[0:2])
 	m.ListenInterval = binary.LittleEndian.Uint16(data[2:4])
+	m.Payload = data[4:]
```

Comment:
Contents field will be set by the following m.Dot11Mgmt.DecodeFromBytes(data, df).